lib/view_component/slot_v2.rb

```diff
@@ -86,8 +86,14 @@ def to_s
     #   end
     # end
     #
-    def method_missing(symbol, *args, &block)
-      @__vc_component_instance.public_send(symbol, *args, &block)
+    if RUBY_VERSION >= "2.7.0"
```

Comment:
Do you think we need to document this difference or is this assumed due to differences between Ruby versions?

Reply:
A bit late to the party, but I think we could use [`ruby2_keywords`](https://ruby-doc.org/core-2.7.3/Module.html#ruby2_keywords-method) here.

I think that would be nice to avoid the `RUBY_VERSION` check.  I believe all that's needed is to change the signature to only accept args, so `def method_missing(symbol, *args, &block)` then below it call `ruby2_keywords(:method_missing) if respond_to?(:ruby2_keywords, true)`.

Thoughts?

Reply:
Oh wow I didn't know about `ruby2_keywords` (or if I did, I forgot about it!) Super handy!